pyvocz/templates/personal-info.html

```diff
@@ -0,0 +1,149 @@
+{% extends "_base.html" %}
+
+{% block bodycontent %}
+<div class="container">
+
+{% if g.lang_code == 'cs' %}
+
+<h1>Pyvo.cz a osobní údaje</h1>
+
+<p>
+    Stránky pyvo.cz zveřejňuje informace o programátorských srazech.
+    Součástí třechto informací jsou údaje o proběhlých a plánovaných
+    přednáškách a přednášejících,
+    a to včetně veřejných <em>audio/video záznamů</em> odborných přednášek
+    zveřejňovaných na stránkách youtube.com.
+</p>
+<p>
+    Veškeré údaje, které pro tento účel spravujeme a zpracováváme,
+    jsou <em>volně k dispozici</em> na webových stránkách
+    <a href="https://pyvo.cz">pyvo.cz</a>;
+    v případě videí na youtube.com.
+</p>
+<p>
+    Jejich součástí mohou být i následující osobní údaje:
+</p>
+<ul>
+    <li>
+        Celé jméno přednášejícího,
+    </li>
+    <li>
+        Audiovideální záznam přednášejícího nebo jiných návštěvníků akce,
+    </li>
+    <li>
+        Kontaktní nebo jiné údaje o přednášejícím, ktreré jsou obsaženy
+        v prezentaci („slajdech“) nebo popisu přednášky,
+        kterou přednášející poskytl organizátorům.
+    </li>
+</ul>
+<p>
+    Tyto údaje zpracovávají i následující společnosti:
```

Comment:
@encukou Co Google? bit.ly/prpylight

Reply:
Nebo srazy.info, a Meetup?
Vypsal jsem služby, kam dávám informace v rámci `pyvo.cz`. Nemyslím si že je v mých silách řešit takhle centrálně organizaci v jednotlivých městech; tohle by se IMO mělo napsat na bit.ly/prpylight zvlášť.

Reply:
Přidal jsme do prylight malou noticku, snad to bude stačit k prokázání dobré vůle.